app/models/events/comment_replied_to.rb

```diff
@@ -3,10 +3,14 @@ class Events::CommentRepliedTo < Event
 
   def self.publish!(reply)
     return unless reply.parent.present?
-    create!(kind: 'comment_replied_to',
-            eventable: reply,
-            user: reply.author,
-            created_at: reply.created_at)
+    event = create!(kind: 'comment_replied_to',
+                    eventable: reply,
+                    user: reply.parent.author,
+                    created_at: reply.created_at)
+
+    ThreadMailer.delay.comment_replied_to(reply.parent.author, event)
```

Comment:
reply.parent.author is right here, but for consistency if nothing else, we'll want to keep the 'user' of the event as the replyer (although I might be able to be convinced otherwise)